.stylelintrc

```
@@ -1,8 +1,11 @@
 {
   "extends": "stylelint-config-recommended",
```

```suggestion
  "extends": "stylelint-config-standard-scss",
```

Ik ben geen frontend expert, maar er is een standaard voor scss, die zou je misschien kunnen gebruiken, gezien ik in dit PR zie dat hier ook scss gebruikt wordt. De package wordt zelfs al toegevoegd aan de package.json

---

I'm no expert either, but did get my hands dirty in upgrading stylelint. These are my findings. By the way, thanks for the (private) link to your current project that rhymes with 'L'Oréal'. That put some things into perspective for me.

The SCSS stylelint implementation at that project differs slightly from this one. The main reason why we do not use the `"extends": "stylelint-config-standard-scss",` option is that the SCSS is generally NOT compliant with that standard. We adhere to the CSS standards :man_shrugging: 

1. I tried fixing the stylesheets to adhere to the SCSS rules. But that ended in a sea of tears as many of the fixed constructs ended up actually breaking the CSS outputted by SASS.
2. The solution implemented is a sort of concession. It does evaluate the code as if being SCSS due to the `"customSyntax": "postcss-scss",` directive. This allows for the nesting and the mixin stuff we do using SCSS. But it lints according to the regular CSS styling rules.